conda-recipe/meta.yaml

```diff
@@ -33,7 +33,7 @@ requirements:
 
 test:
   requires:
-    - scipy  # [linux]
+    - pytest
```

Comment:
Why scipy is removed?

Reply:
pytest can runs without it
Return back?

Reply:
Did you run it with dpnp?

Reply:
@1e-to update to `main` please.